stable/nginx-ingress/README.md

```diff
@@ -1,4 +1,7 @@
-# nginx-ingress
+# DEPRECATED - nginx-ingress
```

Comment:
Please see https://github.com/helm/charts/blob/master/PROCESSES.md for steps to deprecate a chart